inst/@sym/solve.m

```diff
@@ -101,11 +120,10 @@
             '    else:'
             '        symbols.append(arg)'
             '#'
-            'if len(symbols) > 0:'
-            '    d = sp.solve(eqs, symbols, dict=True)'
-            'else:'
-            '    d = sp.solve(eqs, dict=True)'
+            'd = sp.solve(eqs, *symbols, dict=True)'
             '#'
+            'if not isinstance(d, (list, tuple)):'
+            '    return d,'
```

Comment:
This command will fail without it:
```
solve(x*x==1, x>0)
```
and the test exist, is the new, check line 251